code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -543,7 +543,7 @@
 	icon_state = "lingarmor"
 	item_flags = DROPDEL
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
-	armor = list("melee" = 40, "bullet" = 40, "laser" = 40, "energy" = 20, "bomb" = 10, "bio" = 4, "rad" = 0, "fire" = 90, "acid" = 90)
+	armor = list("melee" = 70, "bullet" = 60, "laser" = 30, "energy" = 40, "bomb" = 10, "bio" = 4, "rad" = 0, "fire" = 50, "acid" = 90)
```

Comment:
Why is this a blackhardsuit level good?

Reply:
because it's something that screams "LETHAL ME" and also reduces your chem regen while costing 20 chems to equip. it's also horribly underused because it's meh compared to your other options which you can just raid a locker for while going around the chemcost/chem slowdown. if this proves to be too good i'll adjust as needed.